src/std/text/csv.ss

```diff
@@ -310,13 +310,18 @@
            (string-any (cut w/opt char-needs-quoting? <>) x))
        #t))
 
-;; Given a list of LINES, each of them a list of fields, and a PORT,
+;; Given a list or lsit of LINES, each of them a list of fields, and a PORT,
```

Comment:
a list of LINES or a list of what? And how are those strings to be interpreted? Or is it always a list of LINES, with a line being either a list of strings or a string? Still, what does a string mean then?

Reply:
A list of csv fields as understood by `text/csv`.